requirements.txt

```diff
@@ -1,12 +1,32 @@
+beautifulsoup4==4.6.3
+certifi==2018.8.24
+chardet==3.0.4
+coreapi==2.3.3
+coreschema==0.0.4
+cssutils==1.0.2
 Django==2.1.1
 django-cors-middleware==1.3.1
 django-extensions==2.1.2
+django-inlinecss==0.1.2
+django-rest-swagger==2.2.0
 djangorestframework==3.8.2
 djangorestframework-jwt==1.11.0
 psycopg2-binary==2.7.5
 gunicorn==19.9.0
+idna==2.7
+itypes==1.1.0
+Jinja2==2.10
+MarkupSafe==1.0
+mock==2.0.0
+openapi-codec==1.3.2
+pbr==4.2.0
+psycopg2==2.7.5
 PyJWT==1.6.4
+pynliner3==0.6
 pytz==2018.5
+requests==2.19.1
+simplejson==3.16.0
 six==1.11.0
-django-rest-swagger==2.2.0
+uritemplate==3.0.0
+urllib3==1.23
 whitenoise==4.1
```

Comment:
You only need to include the main dependency that installs  all other sub-dependencies in this case you could have only  included `django-inlinecss==0.1.2`

Reply:
Have you checked out this comment??

Reply:
@gitaumoses4 ??

Reply:
Let me push the changes for this review. Thanks @esirK